public.Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM python:3
+FROM python:3-slim
```

Comment:
Asking for my own understanding, is the gist that `python:3-slim` makes this all faster? Any place where we can read more about it, I see the Airflow option uses another option `slim-stretch` and want to understand more about what this all does.  I did a Google search but I'm getting more hits about something referred to as `alpine`?

Reply:
As I’ve understood the base language `python`, `node` etc. Docker images are based on the Debian distribution. The more info you add to the version part after `:` the more specific you get `-slim` are trimmed down versions, but still based on Debian (examples are that they strip out base OS Python/Perl? etc versions and just add the one you give as “pinned down”). `3` is “current” Python 3, but not as specific as 3.6 etc. `-stretch`, `-buster` etc. reference Debian versions. 

`alpine` is different though, as it’s based on the Alpine distribution. It’s generally a bit smaller than Debian slim - but comes with it’s own Musl lib, so didn’t think it was worth the risk. Recently came across  https://pythonspeed.com/articles/base-image-python-docker-images/ which IMO generally sounds sensible and reflects my experiences with Docker quite well  (He’s also got other posts - mostly pythonic though ..)

I simply opted for `-slim` to reduce size, and thus (mostly) reduced waiting for download. For example with the search repo the uncompressed size is ~225 vs +900 MB. I didn’t add further version pinning down restrictions for now - but we can certainly discuss 

*(of interest for you and @danwom +other frontend folks I’m considering if it’s now time to bump Node to `-10` On what I’ll push after discussion has landed on the first two PRs ... but since you’re better qualified in that end this is the perfect time/place to ask 🙂 )*

Reply:
I see. `-slim` seems good enough! 

Regrading bumping to `Node 10` -- let's continue the discussion in the slack channel if you want.

Reply:
Sure, that thing is certainly worth opening up to to the wider community!